lib/src/data/response/article_response.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:allthenews/src/data/response/media_response.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'article_response.g.dart';
+
+@JsonSerializable()
+class ArticleResponse {
+  @JsonKey(name: 'id')
+  final int id;
+
+  @JsonKey(name: 'url')
```

Comment:
zastanawiam się czy warto dodawać JsonKey w przypadku kiedy nazwa pola jest identyczna z kluczem w odpowiedzi 🤔 Jedyna zaleta to chyba jak ktoś się będzie bawił w refaktor? Ale z drugiej strony to jest tylko warstwa deserializacji i w sumie nie wiem czy mnie przekonuje, że ktoś coś może zmienić nieumyślnie. Tak głośno myślę :D Osobiście raczej bym zbędne adnotacje pousuwał :P

Reply:
wyszedłem z założenia, że przejrzyściej to wygląda, kiedy mamy wszędzie adnotacje, a nie tylko na niektórych polach, ale w sumie sporo ich się powtarza, mogę usunąć